src/NHibernate/Impl/StatelessSessionImpl.cs

```diff
@@ -351,7 +363,7 @@ public override bool IsOpen
 
 		public override FlushMode FlushMode
 		{
-			get { return FlushMode.Commit; }
+			get { return FlushMode.Always; }
```

Comment:
This is more consistent with the stateless semantic of a stateless session, and also with:
https://github.com/nhibernate/nhibernate-core/blob/07ecb9f25534d3894dfba6a3a8e8f22932bd06cf/src/NHibernate/Impl/SessionFactoryImpl.cs#L1605
(That is in the stateless session builder.)